web/themes/joinup/prototype/sass/_mixins.sass

```diff
@@ -72,3 +72,7 @@
     -ms-transform: translate(0, -50%)
     transform: translate(0, -50%)
 
+// Margin for form elements
+=form-margin
```

Comment:
Please improve a bit:
- you can make the size a parameter
- the naming can be more generic

You can get inspiration from: https://github.com/digitaledgeit/sass-spacing